include/game.h

```diff
@@ -1953,6 +1953,21 @@ enum AluTimers {
 #define NUM_HORIZONTAL_SENSORS 4
 #define NUM_VERTICAL_SENSORS 7
 
+typedef enum {
+    VRAM_FLAG_NONE = 0,
+    TOUCHING_GROUND = 1 << 0,
+    TOUCHING_CEILING = 1 << 1,
+    TOUCHING_WALL_RT = 1 << 2,
+    TOUCHING_WALL_LT = 1 << 3,
+    IN_AIR = 1 << 5,
+    VRAM_FLAG_UNK_40 = 1 << 6,
+    VRAM_FLAG_UNK_400 = 1 << 10,
+    TOUCHING_CEILING_SLOPE = 1 << 11,
+    TOUCHING_SLIGHT_SLOPE = 1 << 12,
+    TOUCHING_RAISING_SLOPE = 1 << 14,
+    TOUCHING_ANY_SLOPE = 1 << 15,
+} VramFlags;
```

Comment:
I had done this earlier too in #1851 

Think @Xeeynamo you had some concerns about making this change?
https://github.com/Xeeynamo/sotn-decomp/pull/1851#pullrequestreview-2406759566

Reply:
Added in #1851